### ISSUE TITLE
remove redundant 2021 mention in x-axis

### ISSUE THREAD

**Issue by TimDaub** [rugpullindex/svg-line-chart#20]
<img width="1123" alt="Screen Shot 2021-09-20 at 18 33 41" src="https://user-images.githubusercontent.com/2758453/134039624-cdcb400a-715a-4fe1-906e-fd7f6703a8cc.png">

See x-axis, it says: "Jan 2021 Feb 2021 Mar 2021 ..." etc. However, it's unnecessary to mention 12 times to the user that this month is in 2021. Once would be enough. Hence, we should remove the year and put it once somewhere.

**Comment by soulofmischief** [comment]
`if labels are shown at right position` currently fails for me, and inspecting the three labels generated in that test reveals that the first label has a negative position value.

**Comment by TimDaub** [comment]
mhh few questions:

- which timezone are you?
- What OS are you using
- What is your nodejs version?

**Comment by soulofmischief** [comment]
Yes, my thoughts were node version or my OS being Linux. I just tried changing around the test dates so I don't think it's timezone related, but that's still a possibility. 

* I'm in PST.
* I'm using Linux. OS is the likely culprit here, I agree.
* NodeJS

I get:

```
l1: { pos: -1.5254237288135595, name: 'Dec 2020' }
l2: { pos: 0, name: 'Jan 2021' }
l3: { pos: 1.576271186440678, name: 'Feb 2021' }

```

**Comment by TimDaub** [comment]
Indeed after changing my Mac's timezone to PDT, I get a similar error:

```bash
npm run test -- --match="*if labels are shown*"

> svg-line-chart@0.3.0 test /Users/user/Projects/node-svg-line-chart
> ava --verbose "--match=*if labels are shown*"


{ pos: -1.5254237288135595, name: 'Dec 2020' } { pos: 0, name: 'Jan 2021' } { pos: 1.576271186440678, name: 'Feb 2021' }
  ✖ if labels are shown at right position
  ─

  if labels are shown at right position

  Value is not `true`:

  false

  › file://test/index_test.mjs:180:9

  ─

  1 test failed
```

<img width="658" alt="Screen Shot 2021-10-19 at 03 32 24" src="https://user-images.githubusercontent.com/2758453/137892783-efa318f7-1ee2-4760-9f6b-35a971489539.png">
